tensorflow/python/ops/map_fn.py

```diff
@@ -191,10 +191,10 @@ def map_fn(fn,
   *rows* of a `SparseTensor`.  If you wish to map a function over the nonzero
   values, then you should use:
 
-    * `tf.SparseTensor(st.indices, fn(st.values), st.dense_shape)`
+    * `tf.sparse.SparseTensor(st.indices, fn(st.values), st.dense_shape)`
       (if the function is expressible as TensorFlow ops)
-    * `tf.SparseTensor(st.indices, tf.map_fn(fn, st.values), st.dense_shape)`
-      (otherwise).
+    * `tf.sparse.SparseTensor(st.indices, tf.map_fn(fn, st.values),
+      st.dense_shape)` (otherwise).
```

Comment:
```suggestion
    * otherwise, use
      ```python
      tf.sparse.SparseTensor(st.indices, tf.map_fn(fn, st.values),
                             st.dense_shape)
      ```
```

And then you update the previous bullet point in a similar style

Reply:
Reformatted both bullet points.